automation_tools/__init__.py

```diff
@@ -584,6 +585,10 @@ def setup_vm_provisioning(interface=None):
     )
     run('yum install -y {0}'.format(' '.join(packages)))
 
+    # Setup avahi
+    manage_daemon('start', 'avahi-daemon')
+    manage_daemon('enable', 'avahi-daemon')
```

Comment:
I was surprised that the systemd unit file (and, presumably, the SysV init script) is named avahi-daemon, not avahid. But I quickly logged in to my home server, and on that machine at least, avahi-daemon is the correct name. I've been schooled. At least it's a lot more sane than `NetworkManager`. :p